NickvisionTubeConverter.Shared/Models/Configuration.cs

```diff
@@ -118,6 +123,7 @@ public Configuration()
         UseAria = false;
         AriaMaxConnectionsPerServer = 16;
         AriaMinSplitSize = 20;
+        SubtitleLangs = CultureInfo.CurrentCulture.TwoLetterISOLanguageName;
```

Comment:
Maybe better `$"{CultureInfo.CurrentCulture.TwoLetterISOLanguageName}, {CultureInfo.CurrentCulture.ThreeLetterISOLanguageName}"` ?

Reply:
Would this cause duplicate subtitles to be downloaded for videos, or no?

Reply:
Just tested, looks good.

Reply:
https://github.com/NickvisionApps/TubeConverter/pull/362/commits/dfaca018731ac8d5cc4133595ee8af2c0fab0935

Reply:
I don't think there can be 2 different subtitle labeled with 2- and 3-letter codes of the same langauge. Also, this will show users that both 2- and 3-letter codes are used, depending on a site.

Reply:
> Also, this will show users that both 2- and 3-letter codes are used, depending on a site.

Or maybe it worth adding a help button in row suffix that opens a popup where this is explained.